frontend/src/App.jsx

```
@@ -71,23 +69,68 @@ function App() {
+                </ProtectedRoute>
+              }
+            />
+            <Route
+              path="/my-prescriptions"
+              element={
+                <ProtectedRoute>
+                  <MyPrescriptionsPage />
+                </ProtectedRoute>
+              }
+            />
+            //{" "}
```

Biome (2.1.2)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)